### PR TITLE
Completed Button component

### PR DESCRIPTION
**Pull request by iliyanalibhai**
I might be having some trouble with the spacing of border and text, but here is what it looks like so far
Screenshot 2024-01-12 at 9 35 12 PM

### CODE REVIEW

File: src/components/Button.jsx [hunk 1]
```diff
@@ -1,5 +1,19 @@
+import React from "react";
+import "tailwindcss/tailwind.css";
+
 const Button = () => {
-  return <div>Button</div>;
+  return (
+    <button
+      className="bg-rishi-white border border-rishi-orange text-rishi-black font-urbanist py-2 px-4 rounded"
+      style={{
+        width: "238px",
```
Comment (pondohoo) [inline src/components/Button.jsx:9]: don't use inline css, add all styles within className

File: src/components/Button.jsx [hunk 2]
```diff
@@ -1,5 +1,19 @@
+import React from "react";
+import "tailwindcss/tailwind.css";
+
 const Button = () => {
-  return <div>Button</div>;
+  return (
+    <button
+      className="bg-rishi-white border border-rishi-orange text-rishi-black font-urbanist py-2 px-4 rounded"
```
Comment (pondohoo) [inline src/components/Button.jsx:7]: can you add border-2 px-6 rounded-md and get rid of py-2?

also make it so when you hover your mouse over the button, the bg turns bg-rishi-orange
https://tailwindcss.com/docs/hover-focus-and-other-states

File: src/components/Button.jsx (hunk 1 above)
Comment (iliyanalibhai) [reply src/components/Button.jsx]: For sure, just deleted and pushed to my branch

**Review by pondohoo — APPROVED** [review]
looks perfect great work

**Pull request merged**